src/Site/wwwroot/favicons/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "releases",
+	"icons": [
+		{
+			"src": "\/favicons\/android-chrome-36x36.png",
+			"sizes": "36x36",
+			"type": "image\/png",
+			"density": 0.75
+		},
+		{
+			"src": "\/favicons\/android-chrome-48x48.png",
+			"sizes": "48x48",
+			"type": "image\/png",
+			"density": 1
+		},
+		{
+			"src": "\/favicons\/android-chrome-72x72.png",
+			"sizes": "72x72",
+			"type": "image\/png",
+			"density": 1.5
+		},
+		{
+			"src": "\/favicons\/android-chrome-96x96.png",
+			"sizes": "96x96",
+			"type": "image\/png",
+			"density": 2
+		},
+		{
+			"src": "\/favicons\/android-chrome-144x144.png",
+			"sizes": "144x144",
+			"type": "image\/png",
+			"density": 3
+		},
+		{
+			"src": "\/favicons\/android-chrome-192x192.png",
```

Comment:
Similar to the comment above, could we remove the `\/` at the beginning of each `src` attribute? Assuming that doesn't break anything. :smiley: